extensions/functions_comparison.yaml

```diff
@@ -215,3 +215,49 @@ scalar_functions:
         variadic:
           min: 2
         return: any1
+  -
+    name: "least"
+    description: >-
+      Returns the smallest value.
+
+      The `null_handling` option specifies how to handle null arguments.  If `ANY_NULL`
+      is specified, the function will return null if any argument evaluate to null.
+      `ALL_NULLS` means the function will return null only if all arguments evaluate to null.
+
+      String comparison is done in lexicographical ordering, one character at a time, from
+      left to right. Uppercase letters are less than lowercase letters.
+    impls:
+      - args:
+          - value: T
+        options:
+          null_handling:
+            values: [ ANY_NULL, ALL_NULLS ]
+        variadic:
+          min: 2
+        return: T|?
+  -
+    name: "least_skip_null"
+    description: >-
+      Returns the smallest value. Skip any null arguments.
+
+      String comparison is done in lexicographical ordering, one character at a time, from
+      left to right. Uppercase letters are less than lowercase letters.
+    impls:
+      - args:
+          - value: T
+        variadic:
+          min: 2
+        return: T
+  -
+    name: "greatest"
+    description: >-
+      Returns the largest value. Only return null if 'all' arguments evaluate to null.
```

Comment:
Should probably document that there is no greatest_skip_null because it behaves the same as greatest.